docs/proyectos-de-codigo-abierto.md

```diff
@@ -6,54 +6,75 @@ sidebar_label: Proyectos de Código Abierto
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-## Tablero de Red de EOSIO
+## Proyectos para la comunidad
 
-<img
-  src="https://raw.githubusercontent.com/eoscostarica/guias.eoscostarica.io/master/static/img/eosio-dashboard-logo.png"
-  style={{ width: "30%" }}
-/>
+ Los siguientes proyectos están dirigidos a la comunidad de desarrolladores.
+
+ * * *
+
+### Tablero de Red de EOSIO
```

Comment:
tablero sera  la mejor palabra?

Reply:
`EOSIO Dashboard` is the project name. It is not needed to translate to Spanish